lua/nvim-web-devicons/icons-light.lua

```diff
@@ -47,6 +47,12 @@ local icons_by_filename = {
     cterm_color = "56",
     name = "Eslintrc",
   },
+  [".eslintignore"] = {
+    icon = "",
+    color = "#4b32c3",
+    cterm_color = "56",
+    name = "Eslintrc",
```

Comment:
Let's make this one unique, maybe

```suggestion
    name = "EslintIgnore",
```

Reply:
yeah, you are right